Plone5

pbauer opened the pull request.

pbauer committed:
update buildout to use Plone 5

pbauer committed:
ask before pulling; enable omelette

pbauer committed:
use plone5-branches

pbauer committed:
use plone5 for collective.geo.geographer

parruc committed:
updated history after ploneconf sprint

pbauer committed:
all extras now use branch master
make robot-tests run in new firefox

gborelli commented:
In the packages that need two different versions to maintain the compatibility with plone4 I've already merged the plone5 branch and used a specific branch for plone4. 
Probably we don't need to pin plone5 branch there.

I've also started to work with Docker for developing, if you are interested. See.: 
- https://hub.docker.com/r/gborelli/plone-cgeo/ 
- https://github.com/gborelli/docker-cgeo-development/

pbauer committed:
make travis test with Plone5

pbauer commented:
Great. I'm working on the robot-tests for now.

pbauer commented:
@gborelli I cannot get the robot-tests of the bundle to work properly. Even with a complely new test-setup (based on bobtemplates.plone) the traverser `++add++` does not work on the site created for the tests so I cannot create any content in robot-tests unless I use python. I real-life it works and seems to be a problem with the dependecies.

pbauer commented:
Also: should we aim to make a new release of collective.geo.bundle that works with plone5 and also pulls in collective.geo.behaviour?

pbauer committed:
new test-setup based on best-practices from bob templates.plone

pbauer committed:
add failing test

pbauer commented:
@gborelli I reworked the test-setup and got a little further with finding the culprit. I added the failing test `collective.geo.bundle.tests.test_setup.TestSetup.test_create_document` that shows that instead of the add-view for documents a instance of `collective.geo.mapwidget.browser.widget.MapWidgets` is returned. I have no real idea what that is actually good for and why it is registered the way it is. Can you please have a look at that?

pbauer committed:
no need for plone.testing = 5.0.0

pbauer committed:
we _really_ don't need 5.0.0

pbauer commented:
See https://travis-ci.org/collective/collective.geo.bundle/jobs/125020544

pbauer commented:
@gborelli ping
